apps/ingest/services.py

```diff
@@ -59,6 +59,10 @@ def create_manifest(ingest):
     # This was giving me a 'django.core.exceptions.AppRegistryNotReady: Models aren't loaded yet' error.
     Remote = apps.get_model('ingest.remote')
 
+    # We only want to set the logo_url from a remote manifest, not the logo file
+    if isinstance(ingest, Remote):
+        manifest.logo = None
```

Comment:
I would only set this `None` if the remote manifest definitely includes a logo. It's a "should" in the IIIF spec, but not a requirement, so we could have the default Readux logo as a fallback.

Reply:
Good point - added a check to make sure the manifest has a logo